packages/rum/src/boot/rum.ts

```diff
@@ -198,6 +206,36 @@ function trackErrors(lifeCycle: LifeCycle, handler: (startTime: number, event: R
   })
 }
 
+function trackManualErrors(
```

Comment:
it could have been nice to put this logic directly in an errorCollection file, to avoid to move it and its test with RUMF-740

Reply:
Done